homeassistant/components/binary_sensor/modbus.py

```diff
@@ -7,7 +7,7 @@
 import logging
 import voluptuous as vol
 
-from homeassistant.components import modbus
+from homeassistant.components.modbus import DOMAIN
```

Comment:
This platform belongs to the binary_sensor domain. Import the modbus domain as another name.